Assets/Scripts/InventoryScripts/Pickupable.cs

```diff
@@ -59,16 +61,54 @@ private void Start()
         _tabbedMenu = TabbedMenu.Instance;
         _playerController = PlayerController.Instance;
         _playerInteractBehavior = _playerController.GetComponent<Interact>();
+
+        //On a delay because player inventory is not always loaded before this script
+        Invoke("ValidateInWorld", 5);
+
+        if (_IsInteractiveOnStart)
+        {
+            MakeInteractive();
+        }
+    }
+
+    /// <summary>
+    /// removes this object from the world if it is already in the players inventory.
+    /// Should only be used for the translation book. 
+    /// </summary>
+    private void ValidateInWorld()
+    {
+        GameObject player = GameObject.FindWithTag("Player");
```

Comment:
I'd prefer we find by player controller component instead of by string/tag. Generally more robust and less prone to user errors